core/src/main/resources/AppConstants.properties

```diff
@@ -180,6 +180,7 @@ xml.namespaceAware.default=false
 xslt.streaming.default=true
 xslt.auto.reload=false
 xslt.bufsize=4096
+xslt.default.version=2
```

Comment:
In the current (unwritten) convention it would seem more logical to use the name 'xslt.version.default'